internal/provider/resource_docker_container_migrators.go

```diff
@@ -908,6 +908,9 @@ func updateV0ToV1PortsOrder(is *terraform.InstanceState, meta interface{}) error
 }
 
 func replaceLabelsMapFieldWithSetField(rawState map[string]interface{}) map[string]interface{} {
+	if rawState == nil {
+		return rawState
```

Comment:
nitpicking: I would return nil, cause one normally expects that if something is equal nil, we afterwards assign a value and return that value. In this case since we are just protecting the rest of the function it would be a little more clear that we don't do anything by returning `return nil`. At least that's for me :)

Reply:
Thank you for your review.
It makes sense.

Reply:
Fixed a092c74